content/docs/using-github-directly/index.mdx

```diff
@@ -0,0 +1,303 @@
+5. Next, type: "`brew install node`" into Terminal and hit ‘return’
+6. Then, you need to install ‘Yarn'. Type "`brew install yarn`" into Terminal, and hit ‘return’.
+
+Now that you have yarn installed on your system, you will be able to use it to install all the packages you need to run DARIAH-Campus locally on your computer. To do this:
+
+1. In Finder, go to the folder in which you cloned your DARIAH-Campus fork.
+2. In Terminal, type "`cd`" to change directory (also note, be sure to add a space after ‘cd’), then
+3. Drag and drop the folder from Finder into Terminal _on the same line as “cd “_. This will automatically write the full path (‘breadcrumb’) to the DARIAH-Campus folder in Terminal.
+4. Then type "`pwd`" (path to working directory) in Terminal and hit ‘return’ This will show the correct breadcrumb trail to the folder to make sure you have the right one.
+5. Type "`yarn install`" into Terminal (remember to hit ‘enter’). Installation may take a while if you haven’t installed this before. There is a progress bar on the bottom of Terminal that shows you how far along you are.
+
+Now that you have all the packages you need to preview DARIAH-Campus, you can start the server and build the site like this:
+
+1. In Terminal, go to your local DARIAH-Campus folder (see previous steps 1, and 3 in the previous section for how to do this)
+2. Type "`yarn develop`" (+ enter). This build the whole site locally and start a server to run it in. This may also take a little while.
+3. Once the build process is complete, you will be able to view DARIAH-Campus in the browser using the link given to you in the terminal, which will look like this: ⠀ `http://localhost:8000/`
+4. Copy the url given in Terminal into your browser, which will give you the DARIAH-Campus home page. _Any saved edits you make to the DARIAH-Campus .mdx files in your editor will automatically be shown in your browser._
+
+### Instructions for Windows users
+
+1. Download node.js from [https://nodejs.org/en/download/](https://nodejs.org/en/download/)
+2. Run node.js installer
+3. Open 'command line' and check node version by typing: `node -v`
+4. Download yarn from [https://yarnpkg.com/lang/en/docs/install/#windows-stable](https://yarnpkg.com/lang/en/docs/install/#windows-stable)
+5. Type `Run yarn installer` (enter)
+6. Check yarn version: `yarn --version`
+7. Install the necessary pacakges by typing `yarn install` (+ enter)
+8. In GitHub desktop, make sure that you have the most up-to-date version of DARIAH-Campus. To do this, follow the instructions above to 'Fetch Origin' and 'Merge' it to your local branch
+9. Once any changes from the DARIAH-Campus 'master' on GitHub has been merged to your local branch, navigate to inside the dariah-campus directory (where the file package.json is located) in your file manager.
+10. Run: `yarn develop`. This will build the entire website and launch a server to run it in on your computer. Once the build process is complete, you can:
```

Comment:
```suggestion
10. Run: `yarn dev`. This will build the entire website and launch a server to run it in on your computer. Once the build process is complete, you can:
```